### ISSUE TITLE
Only process src for script tags on top-level?

### ISSUE THREAD

**Issue by shirotech** [kaisermann/svelte-preprocess#77]
Basically for my markup

I have this:

```html
<script>
  export let name;
  export let description;
</script>

<!DOCTYPE html>
<html lang="en">
<head>
  <title>{name}</title>
  <meta content={description} name="description"/>
</head>
<body>
  <div>Test 1</div>
</body>
<script src="some-file.js"></script>
</html>
```

The expected behaviour is the 2nd script tag or a script tag that is not on the top-level should disregard any processing and leave the contents as literal intact.

The current behaviour is the src is being read and parsed which gives an error `[!] (plugin svelte) Error: ENOENT: no such file or directory`

https://github.com/kaisermann/svelte-preprocess/blob/c12637643f721ea2897d71a0e4b6469422b3ca64/src/utils.js#L43

Would be good if this can be fixed easily. Thanks.

**Comment by kaisermann** [comment]
To fix this I'd need to enforce a `./` at the beginning of actual local includes. In theory, this can be a breaking change, but I don't know if people are actually including local files without prepending the `./` 🤔.

**Comment by kaisermann** [comment]
@shirotech Fixed on `v3.1.2` 🎉 Thanks for your feedback 😁

**Comment by shirotech** [comment]
@kaisermann Actually people can still use relative urls in their src and its valid uri for their js file. Was thinking if it can check for file exist, it file does not exist then ignore reading it?

**Comment by kaisermann** [comment]
Oh, you're right 😭. I was trying to prevent checking if the file exists assuming the exception would be the `console.error` itself for the developer, but yeah that makes sense. Wait a sec 🕐

**Comment by kaisermann** [comment]
Released on `v3.1.4`, my bad! 🙌

**Comment by shirotech** [comment]
Nice one, thanks for looking into this, really appreciate it. Thanks.
